app/waf/test/acceptance/service.controller.acceptance.ts

```diff
@@ -9,8 +9,9 @@ import {setupApplication, teardownApplication} from '../helpers/test-helper';
 import {
   givenEmptyDatabase,
   givenServiceData,
-  createServiceObjectWithoutID,
+  createServiceObject,
```

Comment:
Currently our API allows to input resource uuid. I suggest we can continue to have this behavior, until we confirm that it is not require any more. Please continue to have two test cases, one input uuid, one does not.

Reply:
In fact, I think we can combine the `createServiceObjectWithoutID` into `createServiceObject` by passing uuid: 
```
createServiceObject({
    'id': uuid()
});
```
No need to declare a new function with `WithoutID` or `WithID`.

Reply:
agree to change it

Reply:
I have changed the code as the way Andrew suggested.